src/routes/index.ts

```diff
@@ -389,6 +390,15 @@ router.route(config.OVERSEAS_ENTITY_REVIEW_URL)
   .get(overseasEntityReview.get)
   .post(overseasEntityReview.post);
 
+router.route(config.UPDATE_REVIEW_OVERSEAS_ENTITY_INFORMATION_URL)
+  .all(
+    authentication,
+    companyAuthentication,
+    navigation.hasOverseasEntity
```

Comment:
What is the previous page to your new page? The due diligence stuff is it?
If that's the case I think your navigation check should be checking if the user has answered/completed the previous page before accessing your new one. So (if previous page is due diligence) you could use

`navigation.checkDueDiligenceDetailsEntered` instead.

Reply:
Previously the due diligence pages were routing to the overseas entity review page, which was using the navigation.hasOverseasEntity check. Should this have been using the navigation.checkDueDiligenceDetailsEntered check or could there have been another reason for this?

Reply:
yeah that probably makes sense because that page displays the overseas entity's details for you to review, so it checks navigation.hasOverseasEntity.
This new one now is just a basic page so I think generally speaking you check the user has answered the previous page to allow them to view this one.